build/lib/release.sh

```diff
@@ -349,14 +320,13 @@ function kube::release::create_docker_images_for_server() {
 
       local oldifs=$IFS
       IFS=","
-      set $wrappable
+      set "$wrappable"
```

Comment:
hmm what is this used for? what are the contents?

Reply:
wrappable is like this:
```
kube-apiserver,k8s.io/debian-base-amd64:v1.0.0
```
it is comma separated, this change is wrong, I will suppress it.